src/accountsdb/buffer_pool.zig

```diff
@@ -19,26 +21,75 @@ comptime {
     _ = offset;
 }
 
+const FrameRef = packed struct(u32) {
+    f_idx: FrameIndex,
+    hit: bool,
```

Comment:
`is_in_cache`?

Reply:
Not exactly *is* in cache, called it `found_in_cache` https://github.com/Syndica/sig/commit/f40689214b7f8a0945d24c5a2847c29af6db0203